packages/framework/src/Model/Sitemap/SitemapDumper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Sitemap;
+
+use League\Flysystem\FilesystemInterface;
+use League\Flysystem\MountManager;
+use Presta\SitemapBundle\DependencyInjection\Configuration;
+use Presta\SitemapBundle\Service\Dumper;
+use Presta\SitemapBundle\Service\DumperInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class SitemapDumper extends Dumper implements DumperInterface
+{
+    /**
+     * @var \League\Flysystem\FilesystemInterface
+     */
+    private $abstractFilesystem;
```

Comment:
please rename to `$filesystem` for consistency

Reply:
I cannot because Dumper has property $filesystem and I would overwrite it